build-index/tagclient/client.go

```diff
@@ -54,6 +58,21 @@ type singleClient struct {
 	tls  *tls.Config
 }
 
+// Request for ListWithPagination.
+type ListRequest struct {
```

Comment:
can we move ListRequest to tagmodels too?

Reply:
sure

Reply:
This is request from consumers of tagclient. Models are interaction between tagserver and tagclient.